src/modules/search-result/search-result.jsx

```diff
@@ -55,7 +54,22 @@ const SearchResult = ({
 
       <MetadataList>
         <MetadataList.Item id={idFor('author')} label={labels.author}>
-          <AuthorList aria-label={labels.author} authors={author} />
+          <ExpandableList aria-label={labels.author}>
+            {author.map((a) => (
+              <ExpandableList.Item
+                itemProp="author"
+                itemScope=""
+                itemType="https://schema.org/Person"
```

Comment:
I am not sure whether the scope should be here or on the link below. I imagine item-props on the lowest level where only text is a child. However, it's my personal assumption, no reason for it behind.

Reply:
I don't know. You may be right. I haven't really studied the microdata. Probably it's worth to put it in some validator 🤔

Reply:
I could not find anything on the schema.org docs. Let's keep it as it is now.

Reply:
But I find, they along with Wikipedia recommend Google's [Rich Search Results testing](https://search.google.com/test/rich-results) tool.